doc/models/iaf.rst

```diff
@@ -1,5 +1,5 @@
-Integrate-and-Fire (IF) neuron models
-==================================================
+Integrate-and-fire (IF) neuron models
```

Comment:
IF -> IaF

Reply:
This should be addressed in #1374

Reply:
Hm, #1374 seems to remove `iaf.rst` .... We just need to keep an eye on it so once we have merged all the PRs, we have the correct end result.